### PR TITLE
Add simple mta for relaying crontab output

### PR DESCRIPTION
**Pull request by pr0totyp**
Added the simple mail mta [msmtp](https://wiki.alpinelinux.org/wiki/Relay_email_to_gmail_(msmtp,_mailx,_sendmail ) to send crontab output via mail.

My idea was to get mail in case borgmatic is failing for some reason.

In case you want to change anything comment my pull request and I change it. Your borgmatic is great but I am just missing the mail feature. Thanks for your great container :)!

### CODE REVIEW

**Comment by pr0totyp** [comment]
I added the symbolic link to replace the default busybox sendmail with msmtp mail client which was very easy to configure. I was using this [Alpine wiki howto](https://wiki.alpinelinux.org/wiki/Relay_email_to_gmail_(msmtp,_mailx,_sendmail ).

I understand your concerns  to add a another layer with RUN and would prefer to use the installed busybox sendmail but i could't find a way to configure the default busybox sendmail via a mountable config file.

On apt based distributions the sendmail config files can be generated with the sendmailconfig command. This package/command is not available in alpine.

**Comment by pr0totyp** [comment]
> Better don't start a new RUN to avoid a new overlay layer.



> I agree with @toastie89 - this should be added within the single run - also why the need to symbolic link it?


DO

File: Dockerfile [hunk 1]
```diff
@@ -29,8 +29,10 @@ RUN apk upgrade --no-cache \
     ca-certificates \
     lz4-libs \
     libacl \
+    msmtp \
     && rm -rf /var/cache/apk/* \
     && chmod 755 /entry.sh
+RUN ln -sf /usr/bin/msmtp /usr/sbin/sendmail
```
[the remark this answers is not in the document]
Comment (b3vis) [reply Dockerfile]: Thanks @JenswBE 🤘🏽

**Comment by pr0totyp** [comment]
Great :)

**Pull request merged**